java/fury-core/src/main/java/org/apache/fury/meta/Encoders.java

```diff
@@ -38,6 +38,8 @@
 public class Encoders {
   public static final MetaStringEncoder PACKAGE_ENCODER = new MetaStringEncoder('.', '_');
   public static final MetaStringDecoder PACKAGE_DECODER = new MetaStringDecoder('.', '_');
+  public static final MetaStringEncoder GENERIC_ENCODER = PACKAGE_ENCODER;
```

Comment:
How about move GENERIC_ENCODER front and make `PACKAGE_ENCODER = GENERIC_ENCODER`

Reply:
Done